data/tests.yml

```diff
@@ -1158,68 +1161,105 @@
       designated: motor_vehicle
 
 - way_id: 898731283
-  comment: how should an odd number of lanes forward/backwards be split without any clues?
+  comment: guess that an odd number of lanes will be divided favouring the forward direction with a warning
+  rust:
+    expect_warnings: true
   tags:
     highway: "road"
     lanes: "3"
     sidewalk: "both"
   driving_side: "left"
-  output:
-    - type: travel
-      designated: foot
-    - type: travel
-      direction: forward
-      designated: motor_vehicle
-    - type: travel
-      direction: forward
-      designated: motor_vehicle
-    - type: travel
-      direction: backward
-      designated: motor_vehicle
-    - type: travel
-      designated: foot
+  road:
+    highway: road
+    lanes:
+      - type: travel
+        designated: foot
+      - type: travel
+        direction: forward
+        designated: motor_vehicle
+      - type: travel
+        direction: forward
+        designated: motor_vehicle
+      - type: travel
+        direction: backward
+        designated: motor_vehicle
+      - type: travel
+        designated: foot
 
 # I didn't look for a real example of this
 - description: "5 lanes"
+  comment: guess that an odd number of lanes will be divided favouring the forward direction with a warning
+  rust:
+    expect_warnings: true
   tags:
     highway: "road"
     lanes: "5"
     sidewalk: "no"
   driving_side: right
-  output:
-    - type: shoulder
-    - type: travel
-      direction: backward
-      designated: motor_vehicle
-    - type: travel
-      direction: backward
-      designated: motor_vehicle
-    - type: travel
-      direction: forward
-      designated: motor_vehicle
-    - type: travel
-      direction: forward
-      designated: motor_vehicle
-    - type: travel
-      direction: forward
-      designated: motor_vehicle
-    - type: shoulder
+  road:
+    highway: road
+    lanes:
+      - type: shoulder
+      - type: travel
+        direction: backward
+        designated: motor_vehicle
+      - type: travel
+        direction: backward
+        designated: motor_vehicle
+      - type: travel
+        direction: forward
+        designated: motor_vehicle
+      - type: travel
+        direction: forward
+        designated: motor_vehicle
+      - type: travel
+        direction: forward
+        designated: motor_vehicle
+      - type: shoulder
+
+- description: assume lanes:forward to be lanes - lanes:backward
+  skip_python: true
+  skip_kotlin: true
+  tags:
+    highway: secondary
+    lanes: 4
+    lanes:backward: 1
```

Comment:
these are interpreted as integers, when they should be strings, that causes the kotlin tests to fail